addon/index.js

```diff
@@ -0,0 +1,62 @@
+import { isEmpty } from '@ember/utils';
+import { get } from '@ember/object';
+import googletag from 'googletag';
+
+
+export function doTargetingForPath(location=window.location) {
+  doTargeting({
+    'host': location.host,
+    'url': location.pathname,
+    'urlSegments': location.pathname && location.pathname.split('/').filter(segment => segment.length > 0)
+  });
+}
+
+export function clearTargetingForPath() {
+  clearTargeting('host');
+  clearTargeting('url');
+  clearTargeting('urlSegments');
+}
+
+export function doTargetingForModels(...models) {
+  models.forEach(model => {
+    let targets = {};
+    let adTargeting = get(model, 'adTargeting');
+    if(adTargeting) {
+      Object.keys(adTargeting).forEach(key => {
+        targets[key] = get(model, adTargeting[key]);
+      });
+    }
+    doTargeting(targets);
+  });
+}
+
+export function clearTargetingForModels(...models) {
+  models.forEach(model => {
+    let adTargetingMap = get(model, 'adTargeting');
```

Comment:
prefer to use the same variable name as `doTargetingForModels`

Reply:
👍